Tasks/Task2/query03.sql

```diff
@@ -0,0 +1,17 @@
+--Query 3
+SELECT 
+    Sc.CustomerID,	
+    P.FirstName + ' ' + COALESCE(P.MiddleName + ' ', '') + P.LastName as CustomerFullName,
+    SOI.SalesOrderID,
+    SOI.OrderDate
+FROM Sales.SalesOrderHeader SOI
+RIGHT JOIN Sales.Customer SC 
```

Comment:
try to use same type of join ( usually it left :) ) so code could be read/undestood easier